editoast/src/main.rs

```diff
@@ -663,6 +672,14 @@ async fn refresh_search_tables(
     Ok(())
 }
 
+#[derive(Debug, Error, PartialEq)]
+pub enum CLIError {
+    #[error("Infrastructure not found, ID: {0}")]
+    InfraNotFound(i64),
+    #[error("Railjson file not found, Path: {0}")]
+    RailjsonFileNotFound(String),
+}
```

Comment:
We could make a more generic error.

```suggestion
#[derive(Debug, PartialEq)]
pub struct CLIError {
   exit_code: i32,
   message: String,
}
```

@leovalais What do you think?

Reply:
In my opinion, a specific error is better, we only use it in the main.

Reply:
I'd name it `CliError` to respect CamelCase but otherwise I agree with @flomonster since it's just meant to replace `eprintln!`s.